src/examples/simple/ERC6551Account.sol

```diff
@@ -78,7 +78,7 @@ contract ERC6551Account is IERC165, IERC1271, IERC6551Account, IERC6551Executabl
         return bytes4(0);
     }
 
-    function supportsInterface(bytes4 interfaceId) external pure virtual returns (bool) {
+    function supportsInterface(bytes4 interfaceId) public view virtual returns (bool) {
```

Comment:
Just to clarify. 
The function can be restricted to pure, of course, but if so, nobody can expand it as a view, while the opposite is true.